app/src/main/java/org/fossasia/openevent/app/module/event/list/EventListFragment.java

```diff
@@ -93,6 +100,37 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         return binding.getRoot();
     }
 
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+        inflater.inflate(R.menu.menu_events, menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.sortByEventName:
+                sortEvents(SORTBYNAME);
+                return true;
+            case R.id.sortByEventDate:
+                sortEvents(SORTBYDATE);
+                return true;
+            default:
+                return super.onOptionsItemSelected(item);
+        }
+    }
+
+    private void sortEvents(int sortBy) {
+        if (sortBy == SORTBYNAME) {
+            getPresenter().sortByName(SORTBYNAME);
+        } else {
+            getPresenter().sortByName(SORTBYDATE);
+        }
```

Comment:
Every logic should be inside presenter. That's why I said to provide criteria as parameter

Reply:
@iamareebjamal I moved the logic in the presenter, but about the two static variables, they are to be used in both Presenter as well as fragment (In OnOptionItemSelected)